ingestion/src/metadata/ingestion/source/database/oracle/queries.py

```diff
@@ -92,7 +92,7 @@
   NULL AS aborted,
   SQL_FULLTEXT AS query_text,
   TO_TIMESTAMP(FIRST_LOAD_TIME, 'yy-MM-dd/HH24:MI:SS') AS start_time,
-  ELAPSED_TIME AS duration,
+  ELAPSED_TIME / 1000 AS duration,
```

Comment:
for oracle and mssql the duration comes in microseconds so need to divide it by 1000 to convert it to milliseconds